config/locales/en/activerecord.yml

```diff
@@ -75,6 +78,9 @@ en:
       sdg/goal:
         one: "goal"
         other: "goals"
+      sdg/manager:
+        one: "SDG Manager"
```

Comment:
"SDG manager" (lowercase `m`) seems to be more consistent with "goal" and "target" :thinking:.